CONTRIBUTING.md

```diff
@@ -0,0 +1,662 @@
+# Contributing to Stylix
+
+This guide provides the information necessary to contribute to stylix.
+
+> [!NOTE]
+> Having Nix installed with nix-command and pipe-operators enabled is required.
+
```

Comment:
AFAIK, the experimental pipe operator is intentionally not used in the code base. Also, `nix` would already complain when the `nix-command` and `flakes` experimental features are not enabled, making this remark somewhat noisy and obvious:

```suggestion
```